### ISSUE TITLE
Set/persist icon color?

### ISSUE THREAD

**Issue by ttntm** [DanielBorgesOliveira/onedrive_tray#24]
Hi, thanks for this cool piece of software first of all.

Got it running on Fedora 35 today, now I'm wondering if there's any way to persist the chosen icon color; it's a little less convenient that it changes back to blue whenever a new login is performed.

A switch for the launch script would be most welcome if you're maybe planning for a future version.

**Comment by julianfairfax** [comment]
This would be nice

**Comment by bforest76** [comment]
@ttntm @julianfairfax You can try the new version published last week-end. It includes some new features and I think that what you expect is in.

**Comment by julianfairfax** [comment]
> @ttntm @julianfairfax You can try the new version published last week-end. It includes some new features and I think that what you expect is in.

It doesn't seem to persist the icon colour though. Maybe I'm using it wrong?

**Comment by ttntm** [comment]
> > @ttntm @julianfairfax You can try the new version published last week-end. It includes some new features and I think that what you expect is in.
> 
> It doesn't seem to persist the icon colour though. Maybe I'm using it wrong?

Same here unfortunately, defaults to blue and does not persist any other selection across sessions.

**Comment by adrianinsaval** [comment]
the icon color selection is persistent now

**Comment by tmode-1960** [comment]
I've installed here from Github a few days ago. Every start the icon is blue, changes are not persistent. Btw works not under Wayland for me.

**Comment by Morikko** [comment]
Same for me, the icon starts with blue and is not persistent after restart. (on Ubuntu 22.04/Gnome)

**Comment by bforest76** [comment]
> Same for me, the icon starts with blue and is not persistent after restart. (on Ubuntu 22.04/Gnome)

I don't understand. My distro is Ubuntu 22.04.1 LTS / GNOME 42.2 and it works perfectly.

**Comment by bforest76** [comment]
> I've installed here from Github a few days ago. Every start the icon is blue, changes are not persistent. Btw works not under Wayland for me.

Can you give details about your distro ?

**Comment by Morikko** [comment]
Yes, quitting the application by the menu saves the color icon. Previously, I was doing my test with `systemctl restart --user onedrive_tray.service`.

**Comment by DanielBorgesOliveira** [comment]
@Morikko update your code with the last commit, compile the program and try the instructions below.

Open a terminal and call the command below:
`tail ~/.config/onedrive_tray/onedrive_tray.conf `

The variable IconColor should change when you change de icon color.
If the IconColor variable is not changing, maybe you have a permission problem.

**Comment by Morikko** [comment]
@DanielBorgesOliveira Yes, I confirm, update is immediate.
